.github/workflows/test.yml

```
@@ -0,0 +1,174 @@
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              body: coverageComment
+            });
+          }
+
+  test-matrix-complete:
+    needs: test
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - name: Check test matrix status
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)